clang/include/clang/Sema/Sema.h

```diff
@@ -1759,6 +1759,34 @@ class Sema final {
     }
   };
 
+  /// Bitmask to contain the list of reasons a single diagnostic should be
+  /// emitted, based on its language.  This permits multiple offload systems
+  /// to coexist in the same translation unit.
+  enum DeviceDiagnosticReason {
```

Comment:
Can this be made an `enum class` or will that not work with `LLVM_MARK_AS_BITMASK_ENUM`?

Reply:
Ok. Looks like it's working with `enum class`. Converted.